src/routes/Apps/App/LogDialogs.tsx

```diff
@@ -503,6 +503,7 @@ class LogDialogs extends React.Component<Props, ComponentState> {
                 }
                 <div>
                     <OF.PrimaryButton
+                        data-testid="create-new-log-dialog"
```

Comment:
The ids should be what the element is not what it's used for. E.g. `logdialogs-button-create`